moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -113,105 +57,75 @@ const NewProjectView = () => {
   /**
    * Persists a new project into the database
    */
-  const handleSubmit = () => {
-    const newSignalError = fromSignalAsset && !Boolean(signal);
-    setSignalError(newSignalError);
+  const handleSave = (formData) => {
+    const {
+      isSignalProject,
+      signal,
+      projectName,
+      projectSecondaryName,
+      description,
+    } = formData;
+
+    const payload = {
+      project_name: projectName,
+      project_name_secondary:
+        projectSecondaryName.length > 0 ? projectSecondaryName : null,
+      project_description: description,
+      // Use potential phase as default
+      moped_proj_phases: {
+        data: [
+          {
+            phase_id: 1,
+            is_current_phase: true,
+            phase_start: new Date(new Date().setHours(0, 0, 0, 0)),
+            is_phase_start_confirmed: true,
+          },
+        ],
+      },
+    };
 
-    if (projectDetails.project_name.trim().length === 0) {
-      setNameError(true);
-    }
-    if (projectDetails.project_description.trim().length === 0) {
-      setDescriptionError({ message: "Required" });
-    } else if (
-      projectDetails.project_description.trim().length >
-      agolFieldCharMax.descriptionString
-    ) {
-      setDescriptionError({
-        message: `Description must be ${agolFieldCharMax.descriptionString} characters or less`,
-      });
-    }
+    if (isSignalProject) {
```

Comment:
This block is an important change. Some of this data preparation used to happen in the SignalAutocomplete component that is now removed. I reused the helpers that helped turn a Knack record into a Moped signal component ready to be inserted.